src/main/java/com/xebialabs/xlrelease/ci/XLReleaseNotifier.java

```diff
@@ -112,7 +111,7 @@ public boolean perform(final AbstractBuild<?, ?> build, Launcher launcher, Build
         if (startRelease) {
             startRelease(release, template, resolvedVersion, deploymentListener);
         }
-
+        deploymentListener.info(Messages.XLReleaseNotifier_releaseLink(getXLReleaseServer().getServerURL() + "#/releases/" + release.getInternalId().replaceAll("/", "-")));
```

Comment:
Please check formatting, also one line is creating url, setting value and logging we should extract url creation to method which can be reused, also setter and logging should not be done in one go.